src/ui/serverJSONStore.js

```diff
@@ -0,0 +1,297 @@
+import { fireEvent } from '../event'
+import { getFavorites } from './actions/favorite'
+import { waitForWindowProp, joinPath } from '../utils'
+import { createStore } from '../store'
+import { reactive, computed } from '@vue/composition-api'
+import Vue from 'vue'
+import UI_CONSTANTS from '../constants/ui'
+
+class ServerJSONStore {
+    id = 'serverJSON'
+    loaded = false
+    oneTimeLoadPresentations = false
+    state = reactive({
+        files: [],
+        uiFiles: computed(() => this.state.files.filter(file => file.shouldShowInUI)),
+        slides: [],
+        config: null,
+        groups: null,
+        appID: null,
+        categories: [],
+        parentCategories: computed(() =>
+            this.state.categories.filter(category => category.parentCategory == UI_CONSTANTS.PARENT_CATEGORY_VALUE)
+        ),
+        subCategories: computed(() =>
+            this.state.categories.filter(category => category.parentCategory != UI_CONSTANTS.PARENT_CATEGORY_VALUE)
+        ),
+        supportEmail: null,
+        deviceName: null,
+        metadata: null,
+        messages: null,
+        appName: null,
+        documentPath: null,
+        presentations: [],
+        customs: [],
+        category: {},
+        allowedIDs: [],
+        systemLang: null,
+        locale: null,
+        userfullname: null,
+        ajaxtoken: null,
+        isCustomerUI: false,
+        batteryLevel: 0,
+        statusBadge: '',
+        syncEvents: []
+    })
+
```

Comment:
can we move locale and systemLang to i18n store?
can we add SystemStore for stuff like batteryLevel etc?
can we add FileStore for files, categories etc?

just splitting this up so we do not have 1 huge store for everything?